src/lib/list/selection-list.ts

```diff
@@ -111,7 +111,7 @@ export class MatListOption extends _MatListOptionMixinBase
   @Input()
   get selected(): boolean { return this.selectionList.selectedOptions.isSelected(this); }
   set selected(value: boolean) {
-    const isSelected = coerceBooleanProperty(value);
+    const isSelected: boolean = coerceBooleanProperty(value);
```

Comment:
I think you misunderstood what I mean... I mean that we have the following signature for selected: `set selected(value: boolean) { ... = coerceBooleanProperty(value); }`, so the `disabled` should follow the same thing: `set disabled(value: boolean) { ... = coerceBooleanProperty(value); }`. 

Basically change the signature of disabled from `set disabled(value: any) {` to `set disabled(value: boolean) {`.

 Did you get it now?

Reply:
Yea, the highlighted section GitHub was showing was:

```
/** Value of the option */
-  @Input() value: any;
```

So I didn't really get what you were talking about. It's changes now and I removed the unnecessary :boolean.

Reply:
Aha, ok. Nice :)

Reply:
Aha, ok. Nice :)

Will you remove this `:boolean` in another commit?

Reply:
That's the second time today I've committed without first hitting the save button. Done.